monkey/infection_monkey/config.py

```diff
@@ -39,6 +40,9 @@ def from_kv(self, formatted_data):
             elif key == 'exploiter_classes':
                 class_objects = [getattr(exploit_import, val) for val in value]
                 setattr(self, key, class_objects)
+            elif key == 'pe_classes':
```

Comment:
What does it check? If we always try to do PE, I don't think we need this check.

Reply:
@Dhayalanb this is yet another legacy of having the PE be configurable. Since you implemented PEs in the dropper, this can't happen

Reply:
Yes, this is not needed, removed it.